client/package.json

```diff
@@ -8,7 +8,7 @@
     "@emotion/styled": "^11.13.0",
     "@mui/material": "^6.1.1",
     "@niivue/niivue": "^0.32.0",
-    "@tanstack/react-query": "^5.59.16",
+    "@tanstack/react-query": "^5.59.19",
```

Comment:
I'm not sure if the package.json should be in the commit

Reply:
I fixed it!